src/components/Product/Product.js

```diff
@@ -39,7 +39,7 @@ function Product() {
     colecction
       .add({
         name: nameProduct,
-        lastDate: data.lastDate,
+        lastDate: new Date(),
```

Comment:
This is not completely correct.

We should default to `new Date()` only if the user didn't add anything in the form. I think about something like:

```
lastDate: data.lastDate || new Date()
```

Reply:
done!!!